apps/classes/models.py

```diff
@@ -38,3 +53,43 @@ class BasicClassInfo(models.Model):
         validators=[MinValueValidator(1), MaxValueValidator(12)],
     )
     description = models.TextField()
+
+
+class BasicClassSection(BasicClassInfo):
+    """
+    The model will need to store
+    - Section - from parent
+    - Session - done
+    - Type - done
+    - Time
+        - Start time
+        - End time
+    - Days - done
+    - Instructor - done
+    - Location
+        - Building
+        - Room #
+
+    Notes:
+    - Maybe implement default values for future semesters when not all values have been assigned.
+    """
+
+    session = models.CharField(
+        max_length=3,
+    )
+    type = models.IntegerField(choices=SectionType.choices)
```

Comment:
Rename to `section_type` since `type()` is a function in python for better clarity

Reply:
done